### PR TITLE
Overhauled readme

### PR DESCRIPTION
**Pull request by DrunkWinter**
Preview: 
https://github.com/DrunkWinter/Simple-YouTube-Age-Restriction-Bypass#simple-youtube-age-restriction-bypass

### CODE REVIEW

**Comment by DrunkWinter** [comment]
Maybe add some other browsers to the list such as Waterfox, etc?

File: README.md [hunk 1]
```diff
@@ -1,36 +1,91 @@
...
+
+All videos will be unlocked automatically.
+
+## Installation
+1. Install a userscript manager of your choice:<br>
+&bull; [Tampermonkey](https://www.tampermonkey.net)<br>
+&bull; [Violentmonkey](https://violentmonkey.github.io/get-it)<br>
+&bull; [Greasemonkey](https://addons.mozilla.org/en-US/firefox/addon/greasemonkey)<br>
+
+2. Install Simple YouTube Age Restriction Bypass by clicking on one of the [downloads](#downloads).
```
Comment (zerodytrash) [inline README.md:65]: Should we rather link directly to the userscript from the dist folder? I think that would be easier. Some users may not understand the anchor link.

**Review by zerodytrash — COMMENTED** [review]
This looks awesome!

> Maybe add some other browsers to the list such as Waterfox, etc?

hmm idk. I think most Waterfox users are aware that Waterfox is based on Firefox and most other browsers are based on chromium and have the same compatibility.

File: README.md (hunk 1 above)
Comment (DrunkWinter) [reply README.md:65]: I thought the downloads section was clear enough that people would know. 
But feel free to change it as you see fit! ;)

**Pull request merged**